normalizers/regex.js

```diff
@@ -220,7 +220,10 @@ export const androidEscapesEncoder = (str, flags = {}) => {
 
 export const androidSpaceCollapser = (parts) => parts.map(p => (p.t === 's' ? { ...p, v: p.v.replaceAll(/[ \f\n\r\t\v\u2028\u2029]+/g, ' ')} : p));
 
-export const doublePercentDecoder = (parts) => parts.map(p => (p.t === 's' ? { ...p, v: p.v.replaceAll('%%', '%')} : p));
+export const doublePercentDecoder = regexMatchingDecoderMaker(
+    'doublePercentDecoder',
+    /(?<percent>%%)/g,
+    (groups) => groups.percent.replace('%%', '%'));
```

Comment:
why not `'%'` ?

Reply:
Changed.  I also made a change to replace % with '\u0025' (but not %%).